guides/common/modules/con_monitoring-hosts-by-using-insights-in-cloud.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="monitoring-hosts-by-using-{insights-id}-in-cloud"]
+= Monitoring hosts by using {Insights} in {RHCloud}
+
+[role="_abstract"]
+You can use {Insights} to diagnose systems and downtime related to security exploits, performance degradation, and stability failures.
+You can use the {Insights} dashboard in the {RHCloud} to quickly identify key risks to stability, security, and performance.
+You can sort by category, view details of the impact and resolution, and then determine what systems are affected.
+
+If you use {Insights} in {RHCloud}:
+
+* You can use {Insights} services online on the {RHCloud}.
+* {Project} forwards information about your hosts through {Insights} client to calculate recommendations.
+* You can view those recommendations both on the {RHCloud}, the *Advisor* service, and in the {ProjectWebUI} on the *{Insights}* > *Recommendations* page.
+* {Project} uploads host inventory reports to the {RHCloud}.
+* You have control over how much information is included in those reports.
+* You cannot use local {insights-iop}.
+
+.Prerequisites
+* For new {Project} hosts, install and configure {Insights} during host registration to {Project}.
+For more information, see xref:Registering_Hosts_by_Using_Global_Registration_managing-hosts[].
+* For hosts already registered to {Project}, install and configure {Insights} on your hosts by using an Ansible role.
+For more information, see xref:deploying-{insights-id}-client-on-registered-hosts_{context}[].
+ifdef::satellite[]
+* If you register your host to a new {ProjectServer}, reregister the host to {Insights} to avoid creating duplicate entries.
+For more information, see {RHDocsBaseURL}red_hat_lightspeed/1-latest/html/client_configuration_guide_for_red_hat_lightspeed/assembly-client-configuring-insights-client#proc-reregistering-system-insights_insights-cg-configuring-client[Re-registering your system with Red Hat Lightspeed].
+endif::[]
+* You can change the default schedule for running `insights-client` by configuring `insights-client.timer` on a host.
+ifdef::satellite[]
+For more information, see {RHDocsBaseURL}red_hat_lightspeed/1-latest/html/client_configuration_guide_for_red_hat_lightspeed/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Lightspeed_.
+endif::[]
+
+.Troubleshooting
+* You can find the logs for all plugins in the `/var/log/foreman/production.log` file.
+* If you have problems connecting to {Insights}, ensure that your certificates are up-to-date.
+Refresh your subscription manifest to update your certificates.
```

Comment:
I don't see a reason to highlight here that production.log contains logs "for all plugins." (It does, but it's not really relevant.)

Outdated certificates are only one thing that can happen. For problems connecting hosts to Insights, may also want to mention
- make sure the host_registration_insights parameter is set to true for the host (link to that part, ofc)
- make sure the host has access to RHEL BaseOS and AppStream repositories so that insights-client is available to install